src/shared-components/indicator/style.ts

```diff
@@ -1,10 +1,11 @@
 import styled from '@emotion/styled';
 
 import { style as TYPOGRAPHY_STYLE } from '../typography';
-import { SPACER } from '../../constants';
+import { SPACER, ThemeColors } from '../../constants';
 
-export const IndicatorContainer = styled.div`
-  background-color: ${({ theme }) => theme.COLORS.error};
+export const IndicatorContainer = styled.div<{ backgroundColor?: ThemeColors }>`
```

Comment:
An informal convention we had in Cycle 7 with Michael was to use ES6 default values for optional parameters but  in case where we need to use the theme, we moved the logic to the component instead of style file. COuld you please refactor similar to LoadingSpinner component to keep consinstency please.

Reply:
Thanks for letting me know! I refactored and moved the theme to the component. See [e24278e](https://github.com/curology/radiance-ui/pull/718/commits/e24278ecb23f6c686131efaef2863d60525771dc)